corehq/apps/registration/views.py

```diff
@@ -532,11 +532,14 @@ def confirm_domain(request, guid=''):
 
 @retry_resource(3)
 def eula_agreement(request):
+    CURRENT_VERSION = '3.0'  # Should be kept in sync with EulaMixin. Should be a const?
```

Comment:
yeah, agreed this should become a constant

Reply:
do you think `corehq/consts.py` would be the right place or maybe `corehq/apps/app_manager/consts.py`?

Reply:
I think it should go in the users app. There are a couple of constants at the top of `users/models.py` (that's something of a pattern for when we're too lazy to make a dedicated constants file), I'd stick it in there.